src/operators.jl

```diff
@@ -5,6 +5,30 @@ using Krylov
 using LinearAlgebra
 using SparseArrays
 
+# Define mappings for default DEC operations that are not optimizable.
```

Comment:
by optimizable do you mean able to be represented by a sparse matrix? The function below is called `default_dec_matrix_generate`, which has matrix in the name.

Reply:
No, for example, the sharp operator is a matrix, but we do not allocate memory for their outputs.

If I recall correctly, the generic term "optimizable" as used in this code base just means that the assignment uses either `.=` syntax or the `mul!` method when being emitted.

One of the things that I tried to do is amend this state of affairs, by drawing a distinction between `operators with special logic that happen to be matrices`, `operators with special logic that do not happen to be matrices`, and `operators with no special logic and we do not care if they are matrices or not`